public/content/lessons/2019/windmills/index.mdx

```diff
@@ -1,9 +1,232 @@
+    show="video"
+/>
+
+The fourth point is where things get interesting. In some places, your windmill will go around the four points as it did with the triangle.
+
+<Figure
+    image="4_outer.png"
+    video="4_outer.mp4"
+    show="video"
+/>
+
+However, if we put it inside that triangle, then it looks like our windmill never hits it.
+
+<Figure
+    image="4_never.png"
+    video="4_never.mp4"
+    show="video"
+/>
+
+Looking back at the problem, it’s asking you to show that for some starting position of the line, not any position, the process will hit all the points infinitely many times.
+
+So for an example like this, if you start with the line going through that troublesome middle point, what happens? And again, at this point you're just playing around, perhaps moving your pencil among dots you’ve drawn on your scratch paper, questions of rigor and proof will come later.
+
+<Figure
+    image="4_img.png"
+    video="4_img.mp4"
+    show="video"
+/>
+
+Here you’d see that your windmill does indeed bounce off all the points as it goes through a cycle, and it ends up back where it started. The worry you might have is that in some large set of points, where some are kind of inside the others, you might be able to start off on the inside, but maybe the windmill process takes the line to the outside, where it will be blocked off from those inner points.
+
+If you play around, and it might take some time to draw out many examples and think this through, you might notice that when the line starts off passing the middle of all your points, it tends to stay there; it never seems to venture off to the outside. But can you guarantee that this always happens? Or rather, can you first make this idea of starting off in the middle more rigorous, and from there, prove that all points will be hit infinitely many times?
+
+<Figure
+    image="middle.png"
+    video="middle.mp4"
+/>
+
+## The solution
+
+As a general problem-solving tip, whenever you have a vague idea that feels productive, find a way to be more exact about what you mean, preferably putting numbers to it, then ask questions about those numbers.
+
+In our example, one way to formalize the idea of a “middle” is to count how many points are on either side of the line. If you give the line some orientation, you can reasonably talk about the left half, say coloring all points to the left of it blue, and the right half, coloring all points to the right of it brown, and what it means for the line to be in the middle is that there are as many blue points and brown points.
+
+For the moment, let’s say the total number of points is odd, and the point the line passes through is colored white. So for example, if there were 11 points, then you’d have five blue ones on the left of the line, and five brown ones on the right of the line, and the one white one at the pivot. The case with an even number of points will be similar, just slightly less symmetric.
+
+<Figure
+    image="introduce_colors.png"
```

Comment:
If we describe the point the line passes through as white, we should make sure it's white in this image.